OpenKeychain/src/main/res/xml/keyserver_sync_adapter_desc.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<sync-adapter xmlns:android="http://schemas.android.com/apk/res/android"
+    android:contentAuthority="org.sufficientlysecure.keychain.debug.provider"
```

Comment:
Make sure that this also works for non-debug builds. Maybe you need to introduce a string in build.gradle